src/test/java/teammates/test/cases/action/StudentProfilePageActionTest.java

```diff
@@ -81,6 +81,7 @@ private void verifyAccountsAreSame(AccountAttributes student,
         PageData data = result.data;
         student.studentProfile.modifiedDate = data.account.studentProfile.modifiedDate;
         student.createdAt = data.account.createdAt;
+        student.studentProfile.gender = data.account.studentProfile.gender;
```

Comment:
Why is this necessary?

Reply:
Here also overwritting is necessary to remove inconsistency of the Gender (that is enum )

Reply:
@wkurniawan07  Sir here we are verifying the whether the student profile gained from the UI that is from UI/Controller/ShowpageResult and the profile from StudentAccountAttributes in which by default the gender is OTHER so this can create the inconsistency

Reply:
So I am using this to remove inconsistency and two pages can be verified

Reply:
I am not convinced. How exactly will the test fail if this line is not there? Unlike `createdAt`/`modifiedDate` which really depend on when the test is run, this field is not.